dcap/types/src/quote3.rs

```diff
@@ -670,6 +676,7 @@ mod test {
         );
     }
 
+    #[cfg(feature = "alloc")]
```

Comment:
SInce these test cases use vec they don't work unless the alloc feature is present, this got missed since tests are often ran with `--all-features`